.mergify.yml

```diff
@@ -7,6 +7,22 @@ pull_request_rules:
       dismiss_reviews:
         approved: true
         changes_requested: false
+  - name: ask to resolve conflict
+    conditions:
+      - conflict
+    actions:
+      comment:
+        message: This pull request is now in conflicts. Could you fix it? 🙏
+  - name: Request for review
+    conditions:
+      - base=master
+    actions:
+      request_reviews:
+        users:
```

Comment:
Not sure about this... should that not be more component based?
We are reviewing PRs quite actively, I think, and adding other reviewers when we expect them to have a look.

Reply:
removed now